.gitignore

```diff
@@ -7,6 +7,7 @@
 .vs/
 .vscode/
 *.log
+*~
```

Comment:
This is how my editor manages backup files. I can't figure out how to make local git ignore them without changing this here.